Assets/scss/components/bg-design.scss

```diff
@@ -0,0 +1,109 @@
+.bg-design{
+    background-color: var(--bg-design-bg-color);
+    width: fit-content;
+    &-container{
+        display: flex;
+        flex-direction: column;
+        margin-inline-start: toRem(50);
+        margin-inline-end: toRem(80);
+        padding-block-start: toRem(100);
+        margin-block-end: toRem(40);
+
+        @include min-screen(sm) {
+            margin: 0 toRem(150);
+            align-items: center;
+        }
+
+        &--title{
+            color: var(--bg-design-title-color);
+            font-size: toRem(48);
+            margin-inline-end: toRem(40);
+           
+         }
+        &--description{
+            color: var(--bg-design-description-color);
+            font-size: toRem(21);
+            margin: toRem(35) 0;
+        }
+        &--button{
+            cursor: pointer;
+            background-color: var(--bg-design-btnbg-color);
+            border: none;
+            color: $white;
+            font-size: toRem(14);
+            padding: toRem(10) toRem(30);
+            margin-inline-end: toRem(300);
+            @include min-screen(sm) {
+                margin-inline-end: 0;
+            }
+        }
+       
+    }
+
+    &-bottom{
```

Comment:
Could potentially use an ::after pseudo-element instead, so you don't need to put the empty div in the HTML.